apps/spruce/src/components/Content/Layout.tsx

```diff
@@ -57,26 +58,36 @@ export const Layout: React.FC = () => {
       <FloatingContent>
         <TaskStatusIconLegend />
         <Feedback />
+        <WaterfallScrollToTop />
```

Comment:
For some reason I feel like this belongs at the top. That may make it more noticeable
```suggestion
        <WaterfallScrollToTop />
        <TaskStatusIconLegend />
        <Feedback />
```